src/main/java/com/productdock/rbc2024/mapper/BookMapper.java

```diff
@@ -15,34 +15,11 @@
 public class BookMapper {
 
```

Comment:
Could you please remove unused imports? You can use a shortcut: `Command + Option + Shift + L`